python-sdk/tests_integration/astro_deploy/master_dag.py

```diff
@@ -17,16 +17,21 @@
 SLACK_CHANNEL = os.getenv("SLACK_CHANNEL", "#provider-alert")
 SLACK_WEBHOOK_CONN = os.getenv("SLACK_WEBHOOK_CONN", "http_slack")
 SLACK_USERNAME = os.getenv("SLACK_USERNAME", "airflow_app")
+IS_RUNTIME_RELEASE = bool(os.getenv("IS_RUNTIME_RELEASE", False))
```

Comment:
Do we need to introduction of a new configuration? Could we infer if there is a runtime image just based on the environment variable `ASTRONOMER_RUNTIME_VERSION`?

Reply:
@tatiana Reason for adding Env Var is to set the schedule of a master DAG  NONE when this DAG is triggered by automation framework(when a new runtime is released), I am adding IS_RUNTIME_RELEASE in astro cloud automation deployment, using ENV ASTRONOMER_RUNTIME_VERSION will not help as this var is in every runtime deployment

Reply:
Ah, I see, thank you very much, @vatsrahul1001 !